modules/nf-core/kmcp/compute/main.nf

```diff
@@ -0,0 +1,37 @@
+process KMCP_COMPUTE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::kmcp=0.9.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/kmcp:0.9.1--h9ee0642_0':
+        'quay.io/biocontainers/kmcp:0.9.1--h9ee0642_0' }"
+
+    input:
+    tuple val(meta), path(sequences)
+
+    output:
+    tuple val(meta), path("tmp/*") , emit: tmp
```

Comment:
Is `tmp` really the name of the output?

Reply:
Also I would recommend also exporting `_info.txt` which is in the directory as a separate thing too (as wella sin the directory)

Reply:
https://nf-co.re/docs/contributing/modules#naming-conventions point 5 :)
Then in the tesdt you update prefix in the nextflow.config